source/reference/config/triggers.txt

```diff
@@ -96,15 +118,26 @@ Database Triggers
 
 Database trigger configurations conform to the base trigger schema with
 additional configuration options that specify which collection to watch and when
-to fire the trigger. The following fields exist in *database* and *deployment* 
-trigger configuration files:
+to fire the trigger. 
+
+Below are two configurations: one configuration is 
+for a trigger with a Function event processor and the other is for a trigger with 
+an AWS EventBridge event processor. The only difference between the two configuraitons
+is the type of ``event_processor`` and the presence of an ``error_handler`` in the trigger
+with AWS EventBridge. The following fields exist in *database* and *deployment* 
```

Comment:
```suggestion
with AWS EventBridge. The following fields exist in *database*
```
(again, i know this is existing text, but shouldn't this only be for database types? what is "deployment" referring to? (if it's the changestream watch source, aren't there collection, database, _and_ deployment options for database triggers?)

Reply:
I'm honestly not sure what deployment triggers refer to and I can't find it anywhere in the docs. Fine with deleting.